schemas/user_request.py

```diff
@@ -100,7 +105,9 @@ class HostelAccomodationViewOut(BaseModel):
     university_name: str
     organisation_code: str
     payment_recognation: str
-    commandant_full_name: str
+    commandant_first_name: str
```

Comment:
remove unnecessary prefixes
Maybe use the embedded key: "commandant" with first, last, and middle names...